Source/WebKit/Platform/SharedMemory.cpp

```diff
@@ -64,13 +65,46 @@ Ref<SharedBuffer> SharedMemory::createSharedBuffer(size_t dataSize) const
 }
 
 #if !PLATFORM(COCOA)
-void SharedMemory::Handle::takeOwnershipOfMemory(MemoryLedger) const
+void SharedMemoryHandle::takeOwnershipOfMemory(MemoryLedger) const
 {
 }
 
-void SharedMemory::Handle::setOwnershipOfMemory(const ProcessIdentity&, MemoryLedger) const
+void SharedMemoryHandle::setOwnershipOfMemory(const ProcessIdentity&, MemoryLedger) const
 {
 }
 #endif
 
 } // namespace WebKit
+
+namespace IPC {
+
+void ArgumentCoder<WebKit::SharedMemoryHandle>::encode(Encoder& encoder, const WebKit::SharedMemoryHandle& handle)
+{
+    encoder << WTFMove(handle.m_handle);
```

Comment:
This seems wrong. If the handle was passed with `const&` then it’s wrong to use `WTFMove` on one of its data members.

Reply:
The handle is mutable so it’s valid. Later on I want to remove that.